Api/Parameter/ObjectParameter.php

```diff
@@ -286,4 +305,21 @@ function ($value, ParameterConstraintViolationCollection $violations) {
 
         return $this;
     }
+
+    /**
+     * @param string $key
+     *
+     * @return Parameter|null
+     */
+    public function getParameter(string $key): ?Parameter
+    {
+        $parameter = $this->parameters[$key] ?? null;
+
+        if ($this->allowedExtraField && $parameter === null) {
+            // all extra fields will be a default Parameter without any constraints
```

Comment:
"All extra fields are converted to default Parameter without any constraints" could be a better way to explain it